test/articledownloadlinkslist.spec.js

```diff
@@ -1,4 +1,10 @@
-let expect = chai.expect;
+const chai = require('chai');
+const sinon = require('sinon');
+
+const expect = chai.expect;
+const spy = sinon.spy;
+
+let utils = require('../assets/js/libs/elife-utils')();
```

Comment:
This isn't used in this file, so don't need to import it.

Reply:
Completed